cli/flags.rs

```diff
@@ -82,6 +85,12 @@ fn add_run_args<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
       Arg::with_name("no-prompt")
         .long("no-prompt")
         .help("Do not use prompts"),
+    ).arg(
+      Arg::with_name("importmap")
+        .long("importmap")
+        .value_name("FILE")
+        .help("Load import map file")
```

Comment:
Can you link to the spec / gist / manual so people can learn more about the file format?

Reply:
```
--importmap <FILE>
    Load import map file
    Specification: https://wicg.github.io/import-maps/
    Examples: https://github.com/WICG/import-maps#the-import-map
```

Caveat - visible only with `deno --help`